42_grml

```diff
@@ -86,6 +86,13 @@ for file in "${ISO_LOCATION}"/*.iso ; do
      fi
 done
 
+indent='        '
+submenu_indent=
+if [ -n "$GRUB_SUBMENU" -a -n "$iso_list" ] ; then
```

Comment:
nitpick: please let's use `"${GRUB_SUBMENU}"`  etc for all variables, so at least any new code uses consistent variable style :)